### DISCUSSION TITLE
Defunct feed source for Great Western Railway, etc. conversion to GTFS

### DISCUSSION THREAD

**Discussion by bamyers99** [transitland/transitland#389] [Q&A] [answered]
https://www.transit.land/feeds/f-gc-rail~delivery~group~planar~gtfs feed source is defunct per https://github.com/planarnetwork/feeds.planar.network/issues/2
Data is available from https://www.raildeliverygroup.com/our-services/rail-data/fares-timetable-data.html
Conversion program is available from https://github.com/planarnetwork/dtd2mysql

**Comment by irees** [comment, marked as answer]
Currently we only fetch and import GTFS data. We're open to indexing GTFS feeds created by others through a conversion process, but converting the data ourselves can be a complex process and requires a lot of local expertise on the source data.
